lib/framework/fixedpoint.h

```diff
@@ -42,7 +42,7 @@ static const float DEG_1 = (float)65536 / 360.f;
 
 static inline WZ_DECL_CONST float UNDEG(uint16_t angle)
 {
-	return angle * (360 / 65536.0f);
+	return angle * (360.f / 65536.0f) * (3.141592f / 180.f);
```

Comment:
To be clearer about why this is happening, maybe define
`#define degreesToRadians(x) x*(3.141592f/180.0f)`
And use it here. (With a comment specifying that this function returns radians because glm requires it.)